Manual/manJHUGenerator.tex

```diff
@@ -1363,7 +1365,13 @@ \section{JHU Generator Coupling Converter (JHUGenLexicon)}
 \end{verbatim}
 
 \section{Release notes}
+\noindent 
+In going from \verb|v7.5.0| to \verb|v7.5.1| (January 24 2022)
 \noindent
+\begin{itemize}
+\item Support for g4 couplings in process 110 and 114 
```

Comment:
Would prefer admission of bugfix. We were using the wrong coupling for the pseudoscalar tensor structure.